lib/__tests__/cli.test.js

```diff
@@ -361,6 +366,22 @@ describe('CLI', () => {
 		expect(output).toBe('Custom formatter reports 1 warning(s).');
 	});
 
+	it('output a message when wrong --globby-options provided', async () => {
+		await cli([
+			`--globby-options=stylelint-test-globby-options`,
+			'--config',
+			fixturesPath('default-severity-warning.json'),
+			fixturesPath('empty-block.css'),
+		]);
```

Comment:
[suggest] We can refactor the code to make it simpler.

```suggestion
		await cli(['--globby-options=wrong']);
```

Reply:
after refactoring this , do we need to update the "expect" code also at L-377 to L-381? @ybiquitous

Reply:
Yes, please also update the line to change the message expectation:

```diff
-'Invalid option "--globby-options". The value "stylelint-test-globby-options" is not valid JSON object.'
+'Invalid option "--globby-options". The value "wrong" is not valid JSON object.'
```